static/app/utils/theme/theme.chonk.tsx

```diff
@@ -295,3 +311,489 @@ const _chonkDarkColors: typeof _chonkLightColors = {
+  // @TODO(jonasbadalic): should this reference a static color?
+  warningFocus: colors.static.gold200, // Not being used
+
+  /**
+   * A color that indicates something is disabled where user can not interact or use
+   * it in the usual manner (implies that there is an "enabled" state)
+   * NOTE: These are largely used for form elements, which I haven't mocked in ChonkUI
+   */
+  disabled: colors.dynamic.grayTransparent300,
+  disabledBorder: colors.dynamic.grayTransparent300,
+
+  /**
+   * Indicates a "hover" state. Deprecated – use `InteractionStateLayer` instead for
+   * interaction (hover/press) states.
+   * @deprecated
+   */
+  hover: tokens.interactive.button.default.background.hover,
+
+  /**
+   * Indicates that something is "active" or "selected"
+   * NOTE: These are largely used for form elements, which I haven't mocked in ChonkUI
+   */
+  active: colors.static.blurple200,
+  activeHover: colors.static.blurple300,
+  activeText: colors.static.blurple400,
+
+  /**
+   * Indicates that something has "focus", which is different than "active" state as it is more temporal
+   * and should be a bit subtler than active
+   */
+  focus: tokens.interactive.outline.default.color,
+  focusBorder: tokens.interactive.outline.default.color,
+
+  /**
+   * Inactive
+   * NOTE: Used in only a few places, but unclear how this would map to chonkUI
+   */
+  inactive: colors.dynamic.grayTransparent300,
+
+  /**
+   * Link color indicates that something is clickable
+   */
+  linkColor: tokens.interactive.link.accent.color.default,
+  linkHoverColor: tokens.interactive.link.accent.color.hover,
+  linkUnderline: tokens.interactive.link.accent.color.default,
+  linkFocus: tokens.interactive.outline.default.color,
+
+  /**
+   * Form placeholder text color
+   */
+  formPlaceholder: colors.dynamic.grayTransparent300,
+
+  /**
+   * Default form text color
+   */
+  formText: colors.dynamic.grayTransparent300,
+
+  /**
+   *
+   */
+  rowBackground: tokens.static.background.primary,
+
+  /**
+   * Color of lines that flow across the background of the chart to indicate axes levels
+   * (This should only be used for yAxis)
+   */
+  chartLineColor: tokens.static.graphic.chart.annotation.axisLabel,
+
+  /**
+   * Color for chart label text
+   */
+  chartLabel: tokens.static.graphic.chart.canvas.lineGrid,
+
+  /**
+   * Color for the 'others' series in topEvent charts
+   */
+  chartOther: colors.dynamic.grayOpaque200,
+
+  /**
+   * Default Progressbar color
+   */
+  progressBar: colors.static.blurple400,
+
+  /**
+   * Default Progressbar color
+   */
+  progressBackground: colors.dynamic.grayTransparent100,
+
+  /**
+   * Overlay for partial opacity
+   */
+  overlayBackgroundAlpha: colors.dynamic.grayTransparent100,
+
+  /**
+   * Tag progress bars
+   */
+  tagBarHover: colors.static.blurple300,
+  tagBar: colors.dynamic.grayTransparent200,
+
+  // @todo(jonasbadalic) should these reference static colors?
```

Comment:
If the new search component is released for 100% of our users, we can consider removing `searchTokenBackground` and `searchTokenBorder`. @malwilley mentioned that these aren't being used for the new search component.